### PR TITLE
Swagger Docs for SM

### PR DESCRIPTION
**Pull request by mphprogrammer**

### CODE REVIEW

**Comment by mphprogrammer** [comment]
@saneshark I’m working around the attachments issue by creating a VCR cassette of an attachment that contains JSON in the body. The spec passes now, and we can go on. If you can find any information to make the spec more authentic I can change the cassette to reflect that.

**Comment by aub** [comment]
@saneshark The problem isn't with Swagger but with apivore, the thing we use to test the spec. I filed an issue against it [here](https://github.com/westfieldlabs/apivore/issues/108). Let's see if they put in a fix soon, and if not then I can fork the thing and work on it. It seems like a pretty big change would be required.

File: app/swagger/requests/messages/folders.rb [hunk 1]
```diff
@@ -41,6 +41,15 @@ class Folders
                 key :'$ref', :Messages
               end
             end
+
+            # TODO: resolve 500 error that should be 400 error
+            response 400 do
```
Comment (saneshark) [inline app/swagger/requests/messages/folders.rb]: @mphprogrammer can you explain this comment you put in? What seems to be the problem here?

**Comment by saneshark** [comment]
@mphprogrammer I think it's about wrapped up IINM? Let's have @aub give it a second pass if so and get it merged.

File: app/swagger/requests/messages/folders.rb (hunk 1 above)
Comment (mphprogrammer) [inline app/swagger/requests/messages/folders.rb]: @saneshark Should have been removed - not an error - mistyped a spec ...

**Comment by mphprogrammer** [comment]
@saneshark @aub This is pretty much wrapped up - putting up once more pr in an hour with some error specs for folders and messages.

File: app/swagger/requests/messages/folders.rb [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+            response 200 do
+              key :description, 'folder messages response'
+
+              schema do
+                key :'$ref', :Messages
+              end
+            end
+
+            # TODO: resolve 500 error that should be 400 error
```
Comment (patrickvinograd) [inline app/swagger/requests/messages/folders.rb]: Is this TODO still valid? If so can you add more detail as it's not clear to somebody else what the fix is.

File: db/schema.rb [hunk 3]
```diff
@@ -17,6 +17,18 @@
...
+    t.json     "data",                               null: false
+    t.datetime "created_at",                         null: false
+    t.datetime "updated_at",                         null: false
+    t.string   "user_uuid",                          null: false
+    t.json     "list_data",          default: {},    null: false
+    t.boolean  "requested_decision", default: false, null: false
+  end
+
+  add_index "disability_claims", ["user_uuid"], name: "index_disability_claims_on_user_uuid", using: :btree
+
```
Comment (aub) [inline db/schema.rb]: I don't think you want this in here.

File: app/swagger/requests/messages/folders.rb (hunk 2 above)
Comment (mphprogrammer) [inline app/swagger/requests/messages/folders.rb]: Sorry - should have been removed, am pushing up the removed comment commit now.

File: db/schema.rb (hunk 3 above)
Comment (mphprogrammer) [inline db/schema.rb]: How did that get there???

Comment (mphprogrammer) [inline db/schema.rb]: @aub I must have run db:migrate instead of db:setup with the old schema. If ok, could you ok the change?

**Pull request merged**